.travis.yml

```diff
@@ -0,0 +1,40 @@
+branches:
+  only:
+    - main
+
+language: rust
+rust:
+  - stable
+
+env:
+  global:
+    - SCCACHE_VERSION=0.2.13
+    - SCCACHE=$HOME/.sccache/bin/sccache
+    - RUSTC_WRAPPER=$SCCACHE
```

Comment:
Should add `- CARGO_INCREMENTAL=0` to disable incremental compilation according to sccache readme.

Reply:
Good catch!